_includes/header.html

```diff
@@ -50,9 +50,9 @@
       <div class="collapse navbar-collapse">
         <ul class="nav navbar-nav">
           <li><a href="{{site.baseurl | prepend:site.url}}/">/home</a></li>
-          <li><a href="{{site.baseurl | prepend:site.url}}/archive.html">/archive</a></li>
-          <li><a href="{{site.baseurl | prepend:site.url}}/tags.html">/tags</a></li>
-          <li><a href="{{site.baseurl | prepend:site.url}}/about.html">/about</a></li>
+          <li><a href="{{site.baseurl | prepend:site.url}}/archive/">/archive</a></li>
+          <li><a href="{{site.baseurl | prepend:site.url}}/tags/">/tags</a></li>
+          <li><a href="{{site.baseurl | prepend:site.url}}/about/">/about</a></li>
```

Comment:
Good spot, thanks! But could you please remove trailing slashes, with them it would be 404.